### PR TITLE
Fixed no-img-icon for contact and account

### PR DESCRIPTION
**Pull request by wachterjohannes**
Following fixes are done in this PR:

* If no avatar is uploaded for a contact or account the correct icon (house or person) is displayed
* Title in all lists (no underline)
* big input in forms (indicates title)
* refactored html of role form
  * replaced god-mode with select all button
  * name full width

__tasks:__

- [ ] gather feedback for my changes

__related PR:__

* https://github.com/massiveart/husky/pull/547

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/list/main.js [hunk 1]
```diff
@@ -28,6 +28,10 @@ define(function () {
         header: function () {
             return {
                 noBack: true,
+
+                title: 'category.categories.title',
+                underline: false,
```
Comment (danrot) [inline src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/list/main.js]: In general: Is the title sometimes underlined, and sometimes not?

File: src/Sulu/Bundle/SnippetBundle/Resources/public/js/components/snippet/list/main.js [hunk 2]
```diff
@@ -50,6 +51,18 @@ define([
...
+        noBack: true,
+
+        title: 'snippets.snippet.title',
+        underline: false,
+
+        toolbar: {
+            buttons: {
+                add: {},
+                deleteSelected: {}
+>>>>>>> fixed title and security form
```
Comment (danrot) [inline src/Sulu/Bundle/SnippetBundle/Resources/public/js/components/snippet/list/main.js]: Fix the conflict properly :wink:

File: src/Sulu/Bundle/WebsiteBundle/Resources/public/js/components/cache/main.js [hunk 3]
```diff
@@ -45,7 +51,7 @@ define([], function() {
             var context = {translate: this.sandbox.translate},
                 template = this.sandbox.util.template(skeleton, context);
 
-            this.html(template);
+            this.$el.html(template);
```
Comment (danrot) [inline src/Sulu/Bundle/WebsiteBundle/Resources/public/js/components/cache/main.js]: Why is that necessary?

File: src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/list/main.js (hunk 1 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/list/main.js]: in lists it should no be underlined but in forms

File: src/Sulu/Bundle/WebsiteBundle/Resources/public/js/components/cache/main.js (hunk 3 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/WebsiteBundle/Resources/public/js/components/cache/main.js]: because `this.html` breaks everything ... we should handle this (:

**Comment by wachterjohannes** [comment]
@danrot fixed (:

**Comment by danrot** [comment]
In the organization list is the title still underlined
